src/FlipCounter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      x: 0
+  
+    };
+  }
+  add = (num) => {
+    this.setState({ x: parseInt(this.state.x) + 1 });
+    console.log("add activated");
+    console.log(this.state.x);
+  };
+ 
+ 
+ 
+  myChangeHandler = (event) => {
+    this.setState({ value: event.target.value });
+    console.log("change activated");
+  };
+
+  mySubmitHandler = (event) => {
+    this.setState({ x: this.state.value });
+    console.log("submit activated");
+    event.preventDefault();
+  };
+
+  render() {
+   console.log(this.state.x)
+   console.log(this.state.value)
+   const arr = [
+      parseInt((this.state.x / 100000) % 10),
+      parseInt((this.state.x / 10000) % 10),
+      parseInt((this.state.x / 1000) % 10),
+      parseInt((this.state.x / 100) % 10),
+      parseInt(this.state.x / 10) % 10,
+      parseInt(this.state.x % 10)
+    ];
+
+
+    
+    return (
+      <div>
+        <p>this is my counter: </p>
+        <h1>
+          <table>
+            <tr>
+              <td id="main">{arr[0]}</td>
```

Comment:
By the HTML specification, only one unique ID is allowed to be in one HTML document.